content/company-info-and-process/communication/decisions.md

```diff
@@ -1,5 +1,10 @@
 # Making decisions at Sourcegraph
 
+TL;DR
+
+- **Find the right owner.** Every decision should be made by the individual who is most directly responsible for the execution and results of the decision.
+- **Avoid slow decisions**. Making a bad decision fast and reverting later is better than making a good decision slowly. Instead of going to your EM to go to their EM to go to their PM, embrace transparency and ask directly in a public channel for others to see and possibly contribute. It should be an exception when it takes longer than a week to make a decision at our company.
```

Comment:
```suggestion
- **Avoid slow decisions**. Making a bad decision fast and reverting later is better than making a good decision slowly. Instead of going to your manager to go to their manager to go to their product manager, embrace transparency and ask directly in a public channel for others to see and possibly contribute. A decision taking longer than one week should be a rare exception.
```